### PR TITLE
Ses autoyast profile

### PR DESCRIPTION
**Pull request by dzedro**
Autoyast profile fixes for SES hardware in Prague

### CODE REVIEW

File: data/ses/hornet.xml [hunk 1]
```diff
@@ -286,7 +286,7 @@
           <partition_id config:type="integer">131</partition_id>
           <partition_nr config:type="integer">4</partition_nr>
           <resize config:type="boolean">false</resize>
-          <size>500M</size>
+          <size>max</size>
```
Comment (kravciak) [inline data/ses/hornet.xml]: max for boot partition?

Comment (dzedro) [reply data/ses/hornet.xml]: yes

Comment (dzedro) [reply data/ses/hornet.xml]: It will be less than 500, doesn't really matter for master

Comment (kravciak) [reply data/ses/hornet.xml]: ok then :) - I thought it's a typo

Comment (dzedro) [reply data/ses/hornet.xml]: No, I used this way with max for /boot on all nodes.
Just to be somehow consistent in all profiles \^\^

**Pull request merged**